### ISSUE TITLE
bitbucket basic not working

### ISSUE THREAD

**Issue by karmendra** [BadChoice/handesk#298]
I tried both basic as well as oauth to connect to bitbucket but both cases I get following error

ForbiddenAccessException
Can't fetch access_token.

I am not sure why is it going to OAuth2Listener.php line 79 when I am using basic driver.

I read another thread you mentioned that you were never able to make it work via oauth, but basic did work. BTW I am using the master branch as of today.

**Comment by BadChoice** [comment]
yeah... I tried but never succeed in using the oauth with bitbucket, so I ended up using user and password
